rclpy/test/test_init_shutdown.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing
+import sys
+import traceback
+
+
+def run_catch_report_raise(func, *args, **kwargs):
+    try:
+        return func(*args, **kwargs)
+    except:
+        print('exception in {0}():'.format(func.__name__), file=sys.stderr)
+        traceback.print_exc()
+        raise
+
+
+def func_init_shutdown():
+    import rclpy
+    rclpy.init()
+    return rclpy.shutdown() is None
```

Comment:
`rclpy_shutdown` indeed raises a RuntimeError if anything goes wrong otherwise returns None on success. So I'll just catch RuntimeErrors instead of testing return value